prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -38,8 +37,9 @@ class RouterCli : CliktCommand(
     printHelpOnEmptyArgs = true,
 ) {
     private val inputSource: InputSource? by mutuallyExclusiveOptions(
+        option("--merge", help = "list of comma-separated files to merge").convert { InputSource.ListOfFilesSource(it) },
         option("--input", help = "<file1>").convert { InputSource.FileSource(it) },
```

Comment:
New --merge takes a list of files, comma separated.   This seemed easier than changing --input to take multiple files, but that would be the more general approach.